smac/intensification/hyperband.py

```diff
@@ -222,6 +219,10 @@ def get_next_challenger(self,
             run_history=run_history,
             repeat_configs=self.sh_intensifier.repeat_configs
         )
+
+        # For debug purposes, set this flag
```

Comment:
For debugging what exactly?

Reply:
Updated the comment to be:
           # For testing purposes, this attribute highlights whether a
           # new challenger is proposed or not. Not required from a functional
           # perspective